src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import setupStore from './redux/store';
+import './index.css';
+import App from './App';
+// import worker from './mocks/browser';
+// // Start the mocking conditionally.
+// if (process.env.NODE_ENV === 'development') {
+//   worker.start();
+// }
```

Comment:
- [ ] Let's try to not have commented unused code in our codebase as they make our code look dirty and unprofessional. It's a good convention to not have commented unused codes. Kindly remove that 👍🏽